R/clean_redcap_long.R

```diff
@@ -155,9 +155,15 @@ distill_nonrepeat_table_long <- function(
   # Below necessary to remove descriptive text fields
   # and to add column to indicate that form is completed
   my_fields <- db_data_long %>%
-    select(all_of(my_fields), paste0(my_form, "_complete")) %>%
+    select(all_of(my_fields),
+           matches(paste0("^",my_form, "_timestamp$")),
+           paste0(my_form, "_complete")) %>%
```

Comment:
Can you delete ll. 163-165 and add after 159

`matches("^redcap_survey_identifier$"),`

?

Reply:
Yes, good call!